simd/src/KokkosSIMD_dummy.cpp

```diff
@@ -0,0 +1,7 @@
+// This file is need in order to get the linker language
```

Comment:
```suggestion
// This file is needed in order to get the linker language
```
Also, you need to fix the indentation.